react/features/stream-effects/blur/JitsiStreamBlurEffect.js

```diff
@@ -43,6 +44,10 @@ export default class JitsiStreamBlurEffect {
         this._outputCanvasElement = document.createElement('canvas');
         this._outputCanvasElement.getContext('2d');
         this._inputVideoElement = document.createElement('video');
+        this._inputVideoCanvasElement = document.createElement('canvas');
+
+        this._maskFrameTimerWorker = new Worker(timerWorkerScript, { name: 'Blur effect worker' });
+        this._maskFrameTimerWorker.onmessage = this._onMaskFrameTimer;
```

Comment:
Is there a reason why this was moved up to the constructor from the `startEffect` ? The idea was to create the Worker only when the effect is enabled and terminate the Worker in `stopEffect`. Does the blur work fine when its disabled and enabled a second time ?

Reply:
Mistake left over from testing. Removed.